.travis.yml

```diff
@@ -1,4 +1,8 @@
 dist: trusty
+sudo: required
+
+addons:
+  chrome: stable
```

Comment:
I would suggest to merge from master. Copying changes manually may lead to merge conflicts.